### ISSUE TITLE
Simple edits to the data frame metadata

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#7339]
@shadrackkibet please check these suggestions and then I wonder who could do them I assume possibly Antoine, as one of the points involves select!

![image](https://user-images.githubusercontent.com/11226469/160766527-b0ea5873-e5b2-4e1d-842c-e43edc359fd4.png)

Suggestions:
1) Move the current Row Count field to before the Column Count.  Call it `Rows`.
2) Call Column Count simply` Columns`.
3) Move the Filename field to the end. So after the current Keys field.
4) Notice I have a select in place and it has changed the number of columns to 6 (from 60).  Please could that therefore be in red, to indicate it is selected.  (This is why I thought, once agreed, that @N-thony might be the best person to do this?)
5) I also have a filter in place and that fact is ignored.  I suggest we leave that as it is for now, simply on the grounds that life is too short.  But happy if you also wanted to change that to 22 and also put it in red.

**Comment by shadrackkibet** [comment]
@rdstern I agree with these changes. I like them. I claim that point (5) should be fixed even if life is too short!

@N-thony Implement as suggested. Point (5) might be tricky but give it a try. If not let me know and I will have a look at it.
